tricity/tricity/spiders/trojmiasto.py

```diff
@@ -7,14 +7,22 @@ class HousesspiderSpider(scrapy.Spider):
     start_urls = ['https://ogloszenia.trojmiasto.pl/nieruchomosci/dom/gdynia/']
 
     def parse(self, response):
-        houses = response.xpath("//div[@id = 'ogl__list__wrap']").get()
+        houses = response.xpath('//div[contains(@id,"ogl-")]').getall()
 
         for house in houses:
             house_item = HouseItem()
-            house_item['price'] = response.xpath("//p[@class = 'list__item__price__value']//text()").get()
-            house_item['price_per_meter'] = response.css('p.list__item__details__info.details--info--price::text').get()
+            house_item['price'] = response.css('p.list__item__price__value::text').get().replace('zł', '')
+            house_item['price_per_meter'] = response.css('p.list__item__details__info.details--info--price::text').get().replace('zł/m', '')
             house_item['url'] = response.css('a.list__item__content__title__name.link').attrib['href']
-            house_item['area'] = response.xpath("//p[@class='list__item__details__icons__element__desc']//text()").extract_first()
-            house_item['number_of_rooms'] = response.xpath('//ul/li[2]/div/p[2]//text()').extract_first()
+            house_item['area'] = response.xpath("//p[@class='list__item__details__icons__element__desc']//text()").get().replace('\n                                            ', '').replace('\n                                    ', '')
+            house_item['number_of_rooms'] = response.xpath('//ul/li[2]/div/p[2]//text()').get().replace('\n                                            ', '').replace('\n                                    ', '')
 
-            yield house_item
\ No newline at end of file
+            yield house_item
+
+        next_page = response.css('span.pages__item__active').get()
```

Comment:
Do przełączania się na nową stronę to musisz pobrać wartość atrybutu HREF z elementu HTML <a href=">.  Tym elementem na stronie jest `>` przełącza na kolejną stronę. Tutaj jak to można zrobić z przykładem
https://stackoverflow.com/questions/21181628/get-href-using-css-selector-with-scrapy.

Jak będziesz miała tą wartość to będzie coś jak "?strona=1", to wtedy musisz to połączyć z URL strony, można to pobrać albo z array  start_urls (linijka nr. 7 ) który jest property klasy HousesspiderSpider, albo w debuggerze można sprawdzić czy obiekt `response` nie ma jakiejś informacji którą można użyć.